source/usage-examples/findOne.go

```diff
@@ -0,0 +1,54 @@
+//Package main will provides a code example showing
+//how to find a document using the MongoDB Go Driver.
+package main
+
+import (
+	"context"
+	"fmt"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"log"
+)
+
+//main finds and prints the lowest rated movie titled "The Room" in the sample dataset.
+func main() {
+
+	// Replace the uri string with your MongoDB deployment's connection string.
+	uri := "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority"
+
+	// Create an empty context
+	ctx := context.TODO()
+
+	// Connect to your MongoDB deployment
+	client, err := mongo.Connect(ctx, options.Client().ApplyURI(uri))
+	if err != nil {
+		panic(err)
+	}
+	// Disconnect Client
+	defer func() {
+		if err = client.Disconnect(ctx); err != nil {
+			panic(err)
+		}
+	}()
+	// Define Collection
+	coll := client.Database("sample_mflix").Collection("movies")
+	var result bson.M
+	// Initialize an empty *FindOneoptions value
+	opts := options.FindOne()
+	// Project only the `title` and `imdb` fields in the returned document
+	opts = opts.SetProjection(bson.M{"_id": 0, "title": 1, "imdb": 1})
+	// Sort matched documents in descending order by rating
+	opts = opts.SetSort(bson.D{{"rating", -1}})
+	// Retrieve your document using a query and your configuration options
+	err = coll.FindOne(ctx, bson.M{"title": "The Room"}, opts).Decode(&result)
+	if err != nil {
+		// ErrNoDocuments means that the filter did not match any documents in the collection
+		if err == mongo.ErrNoDocuments {
+			return
+		}
+		log.Fatal(err)
+	}
+	// Print your matched document
+	fmt.Printf("%v\n", result)
```

Comment:
Since you're not providing any message, why not not just use `fmt.Println(result)`?

Reply:
Thank you for the note. I agree this is the better function to use in the code example. 

Fix: https://github.com/mongodb/docs-golang/pull/2/commits/207e7977f4fcf49826ea67887e281efc7c002c39

I will try be more critical of the functions I choose for code snippets going forward.

Reply:
I'm fine with the solution you committed, but alternatively, you could add explanatory text in which case Printf would make more sense.